src/fluree/crypto/encodings.cljc

```diff
@@ -202,7 +204,9 @@
   "Decode a X9.62 encoded public key from hex"
   ^ECPoint
   [public-key curve]
-  (assert (#{"02" "03" "04"} (subs public-key 0 2)) "X9.62 encoded public key must have a first byte of 0x02, 0x03 or 0x04.")
+  (assert (#{"02" "03" "04"}
```

Comment:
I realize this has always been an assert, just thinking it probably better to throw an ex-info instead of an AssertionError.